python/cudf/bindings/csv.pyx

```diff
@@ -249,6 +249,7 @@ cpdef cpp_write_csv(
     false_value = 'False'.encode()
     csv_writer.false_value = false_value
     csv_writer.include_header = header
+    csv_writer.rows_per_chunk = 10240000
```

Comment:
Pandas defaults to writing all the rows in a single chunk, but as long as this still writes to a single file I think it's okay for our default chunksize to differ here. This still writes to a single file regardless of the number of chunks, correct?

Reply:
yes, that is correct